data_structures/linkedlist/cyclically_linkedlist/readme.md

```diff
@@ -1,12 +1,14 @@
-# Linked Cyclic List
+# Cyclic Linked List
 ***
 ## What is it?
 ***
 Loop lists are single or doubly-linked lists that chase their own tail:
-A points to B, B points to C, C points to D, and D points to A. 
+A point to B, B point to C, C point to D, and D point to A.
```

Comment:
```suggestion
A points to B, B points to C, C points to D, and D points to A.
```

Reply:
The previous one was grammatically correct 😄

Reply:
:D Oh really? I used Grammarly, which shows me that `points to` might be incorrect, so I thought `point to` might be correct. Sorry about that.